migrations/2020-10-07-234221_fix_fast_triggers/down.sql

```diff
@@ -220,14 +221,36 @@ BEGIN
             post_aggregates_view
         WHERE
             id = NEW.post_id;
-        -- Force the hot rank as zero on week-older posts
+        -- Update the comment hot_ranks as of last week
+        UPDATE
```

Comment:
These are really old migrations that are updating tables and views that aren't used anymore, whats going on?